### ISSUE TITLE
Issue with boundary integrals in 1d

### ISSUE THREAD

**Issue by jw3126** [gridap/Gridap.jl#368]
From gitter:
```julia
using Gridap

model= CartesianDiscreteModel((0,1), 10)
V = TestFESpace(reffe=:Lagrangian, order=1, model=model, valuetype=Float64)
    
u_e(pt) = -1/2*pt[1]^2

F(u, v) = u ⊙ v - u_e ⊙ v
B(u, v) = u ⊙ v - u_e ⊙ v
    
trian = Triangulation(model)
quad = CellQuadrature(trian, 2)
t_inner = FETerm(F, trian, quad)
    
btrian = BoundaryTriangulation(model, "boundary")
bquad = CellQuadrature(btrian, 2)
t_bdry = FETerm(B, btrian, bquad)
    
U = TrialFESpace(V)
op = FEOperator(U,V,t_inner, t_bdry)
sol = solve(op)
```
```
MethodError: zero(::Type{Union{}}) is ambiguous. Candidates:
```

**Comment by fverdugo** [comment]
About to be fixed in PR #393

**Comment by fverdugo** [comment]
Closed via PR #393
